### PR TITLE
Add indexedBillingTeamMember feature flag for galley

### PR DESCRIPTION
**Pull request by akshaymankar**
Depends on https://github.com/wireapp/wire-server/pull/1081, but should be merged before that.
Fixes zinfra/backend-issues#1296

### CODE REVIEW

File: charts/galley/templates/configmap.yaml [hunk 1]
```diff
@@ -47,6 +47,9 @@ data:
       maxFanoutSize: {{ .settings.maxFanoutSize }}
       {{- end }}
       conversationCodeURI: {{ .settings.conversationCodeURI | quote }}
+      {{- if .settings.indexedBillingTeamMembers }}
```
Comment (tiago-loureiro) [inline charts/galley/templates/configmap.yaml:50]:
```suggestion
      {{- if .settings.enableIndexedBillingTeamMembers }}
```

Comment (tiago-loureiro) [reply charts/galley/templates/configmap.yaml:50]: Think it was a typo?

**Comment by akshaymankar** [comment]
The integration tests I wrote in https://github.com/wireapp/wire-server/pull/1081 assume that this flag is true by default in galley. So, this PR is a blocker for the one in wire-server.

@tiago-loureiro Can you please approve this if it looks ok?

**Comment by tiago-loureiro** [comment]
:shipit:

**Pull request merged**